deploy/backplane/10-cee-read.ClusterRole.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-cee-read-all
```

Comment:
I'd like to see use of role aggregation.  Notes on how it works here:
https://github.com/jewzaam/demo-role-aggregation

Basically you set an aggregationRule on this clusterrole that requires some label.  Then we can add that label to the osd-readers-aggregate clusterrole:
https://github.com/openshift/managed-cluster-config/blob/master/deploy/rbac-permissions-operator-config/05-osd-readers.ClusterRole.yaml